plexpy/notifiers.py

```diff
@@ -1417,3 +1437,68 @@ def return_config_options(self):
                          ]
 
         return config_option
+
+class TELEGRAM(object):
+
+    def __init__(self):
+        self.enabled = plexpy.CONFIG.TELEGRAM_ENABLED
+        self.bot_token = plexpy.CONFIG.TELEGRAM_BOT_TOKEN
+        self.chat_id = plexpy.CONFIG.TELEGRAM_CHAT_ID
+
+    def conf(self, options):
+        return cherrypy.config['config'].get('Telegram', options)
+
+    def notify(self, message, event):
+        if not message or not event:
+            return
+
+        http_handler = HTTPSConnection("api.telegram.org")
+
+        data = {'chat_id': self.chat_id,
+                'text': message.encode("utf-8")}
```

Comment:
I think I understand.

```javascript
elif config_id == 13:
          telegramClient = TELEGRAM()
          telegramClient.notify(message=body, event=subject)
```

I pass the subject as event, so is this what you mean?

```javascript
data = {'chat_id': self.chat_id,
    'text': event + ': ' + message.encode("utf-8")}
```